dotnet/src/Connectors/Connectors.HuggingFace/TextEmbedding/HuggingFaceTextEmbeddingGenerationService.cs

```diff
@@ -112,6 +113,7 @@ private async Task<IList<ReadOnlyMemory<float>>> ExecuteEmbeddingRequestAsync(IL
         using var httpRequestMessage = HttpRequest.CreatePostRequest(this.GetRequestUri(), embeddingRequest);
 
         httpRequestMessage.Headers.Add("User-Agent", HttpHeaderValues.UserAgent);
+        httpRequestMessage.Headers.Add(HttpHeaderNames.SemanticKernelVersion, Assembly.GetExecutingAssembly().GetName().Version.ToString());
```

Comment:
It's possible to get current assembly in more efficient way. Here are remarks about `GetExecutingAssembly` method:
<img width="546" alt="image" src="https://github.com/microsoft/semantic-kernel/assets/13853051/872058ec-32eb-4b17-9e58-0eaf12680ef5">